clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractHeartbeatRequestManager.java

```diff
@@ -467,12 +441,12 @@ static class HeartbeatRequestState extends RequestState {
         private long heartbeatIntervalMs;
 
         public HeartbeatRequestState(
-            final LogContext logContext,
-            final Time time,
-            final long heartbeatIntervalMs,
-            final long retryBackoffMs,
-            final long retryBackoffMaxMs,
-            final double jitter) {
+                final LogContext logContext,
+                final Time time,
+                final long heartbeatIntervalMs,
+                final long retryBackoffMs,
+                final long retryBackoffMaxMs,
+                final double jitter) {
```

Comment:
indentation off?

Reply:
Personally, I disagree in this case. It means the parameters are not indented to exactly the same level as the statements that follow aiding readability.

Reply:
Totally! I didn't see it along with the following statements, much better indeed.